src/components/Expandable.js

```diff
@@ -0,0 +1,45 @@
+import { html } from 'htm/preact';
+import { useState, useRef, useEffect } from 'preact/compat';
+
+const chevronRight = new URL('../assets/icons/chevron-right.svg', import.meta.url).href;
+const chevronDown = new URL('../assets/icons/chevron-down.svg', import.meta.url).href;
+
+export function Expandable(props) {
+  const [expanded, setExpanded] = useState(false);
```

Comment:
do we want to be consistent with use of hooks / classes?

Reply:
I figured here it would be okay to use hooks because its just a single component that wont use any class components as children. Although it may be valuable to be able to do something like

```js
<${Expandable}>
  <${SomeComponent} />
<//>
```

So maybe I could refactor it to a class. What do you think?

Reply:
I don't think class or function changes how composition behavior works?

Reply:
you cant use class components inside a hook component

Reply:
💥